tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/PartialViews.spec.ts

```diff
@@ -0,0 +1,240 @@
+    // Act
+    await umbracoApi.partialView.ensureNameNotExistsAtRoot(partialViewName);
+
+    await page.locator('umb-tree-item', {hasText: 'Partial Views'}).getByLabel('Open actions menu').click({force: true});
+    await page.getByLabel('New empty partial view').click();
+    await page.getByLabel('template name').fill(partialViewName);
+    // TODO: Remove this timeout when frontend validation is implemented
+    await page.waitForTimeout(1000);
+    await page.getByLabel('Save').click();
+
+    // Assert
+    expect(await umbracoApi.partialView.nameExistsAtRoot(partialViewFileName)).toBeTruthy;
+    // TODO: when frontend is ready, uncomment this to verify the new partial view is displayed under the Partial Views section 
+    //await expect(page.locator('umb-tree-item', {hasText: 'Partial Views'}).getByLabel(partialViewFileName)).toBeVisible();
+    // TODO: when frontend is ready, verify the notification displays
+    
+    // Clean
+    await umbracoApi.partialView.ensureNameNotExistsAtRoot(partialViewFileName);
+  });
+
+  test('can create a partial view from snippet', async ({page, umbracoApi}) => {
+    const partialViewName = 'TestPartialViewFromSnippet';
+    const partialViewFileName = partialViewName + ".cshtml";
+    // Act
+    await umbracoApi.partialView.ensureNameNotExistsAtRoot(partialViewFileName);
+
+    await page.locator('umb-tree-item', {hasText: 'Partial Views'}).getByLabel('Open actions menu').click({force: true});
+    await page.getByLabel('New partial view from snippet...').click();
+    await page.getByLabel('Breadcrumb').click();
+    await page.getByLabel('template name').fill(partialViewName);
+    // TODO: Remove this timeout when frontend validation is implemented
+    await page.waitForTimeout(1000);
+    await page.getByLabel('Save').click();
+    
+
+    // Assert
+    expect(await umbracoApi.partialView.nameExistsAtRoot(partialViewFileName)).toBeTruthy;
+    // TODO: when frontend is ready, uncomment this to verify the new partial view is displayed under the Partial Views section
+    //await expect(page.locator('umb-tree-item', {hasText: 'Partial Views'}).getByLabel(partialViewFileName)).toBeVisible();
+    // TODO: when frontend is ready, verify the notification displays
+    
+    // Clean
+    await umbracoApi.partialView.ensureNameNotExistsAtRoot(partialViewFileName);
+  });
+
+  
+test('can update a partial view name', async ({page, umbracoApi}) => {
+    const partialViewName = 'TestPartialView';
+    const partialViewFileName = partialViewName + ".cshtml";
+    const updatedPartialViewName = 'TestPartialViewUpdated';
+    const updatedPartialViewFileName = updatedPartialViewName + ".cshtml";
+    // Arrange
+    await umbracoApi.partialView.ensureNameNotExistsAtRoot(partialViewFileName);
+    await umbracoApi.partialView.create(partialViewFileName, "@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage\r\n", "/");
+
+    //Act
+    gotoPartialViewDetail(page, partialViewFileName);
```

Comment:
Usages of gotoPartialViewDetail should be awaited
```suggestion
    await gotoPartialViewDetail(page, partialViewFileName);
```